src/common/function/FunctionManager.cpp

```diff
@@ -41,6 +42,7 @@ std::unordered_map<std::string, Value::Type> FunctionManager::variadicFunReturnT
     {"concat_ws", Value::Type::STRING},
     {"cos_similarity", Value::Type::FLOAT},
     {"coalesce", Value::Type::__EMPTY__},
+    {"_any", Value::Type::__EMPTY__},
```

Comment:
the new added _any function, is it callable from query?

Reply:
Can't.

Reply:
If I understood correctly, it seems to me that you mean `_all`, not `_any`. If there are 10 edge types, all of which has the same prop: `prop_x`, the semantics of `any` could be satisfied if only 1 such `prop_x` is found from all 10 edge types. `all` means making sure and finding all `prop_x` from all 10 edge types. I think you mean the latter one.